server/wavelength/modules/economy/index.js

```diff
@@ -0,0 +1,124 @@
+// Used with nef
+
+'use strict';
+
+const FS = require(('../../../../.lib-dist/fs')).FS;
+const DB_PATH = 'config/database'; // path to your database folder
+
+module.exports = class Economy {
+	constructor(name, dbName) {
+		this.name = name;
+		this.dbName = dbName;
+		this.db = Db[dbName];
+
+		this.transactionLogs = FS('logs/transactions.txt');
+		this.tLogs = this.transactionLogs.createAppendStream();
+	}
+
+	grammarizeName(value) {
```

Comment:
```suggestion
	/**
	 * @param {number} value
	 */
	grammarizeName(value) {
```
Type documentation is needed for these functions.